page.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>KMarketer</title>
+    <meta charset="UTF-8">
+    <style>
+        body {
+            background-color: rgba(0, 0, 0, 0.1);
+            background-image: url("background.png");
+            background-size: cover;
+            background-attachment: fixed;
+            background-position: center;
+            color: white;
+            text-align: center;
+        }
+
+        h1,
+        h4 {
+            font-family: "Lucida Console", "Courier New", monospace;
+        }
+
+        h1 {
+            margin-top: 330px;
+            width: 100%;
+            font-size: 70px;
+        }
+
+        h4 {
+            font-size: 25px;
+
+        }
+
+        p {
```

Comment:
it's time to get used with the classes. So please avoid using tags as selectors. 
Just imagine that you will add more p and h4 on your page. 

Suggestion:
create a div wrapper for your top banner (thee image on bg, nav and title usually are called "top-banner") add a specific class to the wrapper.

And use this wrapper class to specify styles for it's children